sdk/maps/maps-timezone-rest/review/maps-timezone.api.md

```diff
@@ -0,0 +1,369 @@
+}
+
+// @public (undocumented)
+export interface GetIanaVersion {
+    get(options?: TimeZoneGetIanaVersionParameters): StreamableMethod<TimeZoneGetIanaVersion200Response | TimeZoneGetIanaVersionDefaultResponse>;
+}
+
+// @public (undocumented)
+export interface GetTimeZoneByCoordinates {
+    get(options: TimeZoneGetTimeZoneByCoordinatesParameters): StreamableMethod<TimeZoneGetTimeZoneByCoordinates200Response | TimeZoneGetTimeZoneByCoordinatesDefaultResponse>;
+}
+
+// @public (undocumented)
+export interface GetTimeZoneByID {
+    get(options: TimeZoneGetTimeZoneByIDParameters): StreamableMethod<TimeZoneGetTimeZoneByID200Response | TimeZoneGetTimeZoneByIDDefaultResponse>;
+}
+
+// @public (undocumented)
+export interface GetWindowsTimezoneIds {
+    get(options?: TimeZoneGetWindowsTimezoneIdsParameters): StreamableMethod<TimeZoneGetWindowsTimezoneIds200Response | TimeZoneGetWindowsTimezoneIdsDefaultResponse>;
+}
+
+// @public (undocumented)
+export interface IanaIdOutput {
+    AliasOf?: string;
+    HasZone1970Location?: boolean;
+    Id?: string;
+    IsAlias?: boolean;
+}
+
+// @public (undocumented)
+export function isUnexpected(response: TimeZoneGetTimeZoneByID200Response | TimeZoneGetTimeZoneByIDDefaultResponse): response is TimeZoneGetTimeZoneByIDDefaultResponse;
+
+// @public (undocumented)
+export function isUnexpected(response: TimeZoneGetTimeZoneByCoordinates200Response | TimeZoneGetTimeZoneByCoordinatesDefaultResponse): response is TimeZoneGetTimeZoneByCoordinatesDefaultResponse;
+
+// @public (undocumented)
+export function isUnexpected(response: TimeZoneGetWindowsTimezoneIds200Response | TimeZoneGetWindowsTimezoneIdsDefaultResponse): response is TimeZoneGetWindowsTimezoneIdsDefaultResponse;
+
+// @public (undocumented)
+export function isUnexpected(response: TimeZoneGetIanaTimezoneIds200Response | TimeZoneGetIanaTimezoneIdsDefaultResponse): response is TimeZoneGetIanaTimezoneIdsDefaultResponse;
+
+// @public (undocumented)
+export function isUnexpected(response: TimeZoneGetIanaVersion200Response | TimeZoneGetIanaVersionDefaultResponse): response is TimeZoneGetIanaVersionDefaultResponse;
+
+// @public (undocumented)
+export function isUnexpected(response: TimeZoneConvertWindowsTimezoneToIana200Response | TimeZoneConvertWindowsTimezoneToIanaDefaultResponse): response is TimeZoneConvertWindowsTimezoneToIanaDefaultResponse;
+
+// @public
+function MapsTimeZone(credential: AzureKeyCredential, options?: ClientOptions): MapsTimeZoneClient;
+
+// @public
+function MapsTimeZone(credential: TokenCredential, mapsAccountClientId: string, options?: ClientOptions): MapsTimeZoneClient;
+
+// @public
+function MapsTimeZone(credential: AzureSASCredential, options?: ClientOptions): MapsTimeZoneClient;
+export default MapsTimeZone;
+
+// @public (undocumented)
+export type MapsTimeZoneClient = Client & {
+    path: Routes;
+};
+
+// @public
+export interface ReferenceTimeOutput {
+    DaylightSavings?: string;
```

Comment:
This should be renamed to `DaylightSavingsName`

Reply:
Maybe I'm using wrong approach, but seems I can't rename it with these directives:

`- from: swagger-document
  where: $.definitions.TimeTransition.properties.DaylightSavings
  transform: |
    $["x-ms-client-name"] = "DaylightSavingsName";

- from: swagger-document
  where: $.definitions.ReferenceTime.properties.DaylightSavings
  transform: |
    $["x-ms-client-name"] = "DaylightSavingsName";`

Reply:
As it turned out we can't modify RCL properties. It would cause mismatch between client code and the server's response (which will result in parsing failure).